app/assets/javascripts/spotlight/pages.js.erb

```diff
@@ -22,8 +22,7 @@ Spotlight.onLoad(function(){
   if (instance.length) {
     var editor = new SirTrevor.Editor({
       el: instance[0],
-      blockTypes:["Heading", "Text", "List","Quote","Iframe", "Video", "Oembed", "Rule", "UploadedItems",
-                 "Browse", "FeaturedPages", "SolrDocuments",  "SolrDocumentsCarousel", "SolrDocumentsEmbed", "SolrDocumentsFeatures", "SolrDocumentsGrid", "SearchResults"],
+      blockTypes: <%= Spotlight::Engine.config.sir_trevor_widgets %>,
```

Comment:
I'm not sure I fully grock what you are proposing. Would this allow `blockTypes` to be just grabbed from the instantiating block type based on the data-attribute?